Safari/Info.plist

```diff
@@ -3,23 +3,23 @@
 <plist version="1.0">
 <dict>
 	<key>Author</key>
-	<string>STUDIOXENIX</string>
+	<string>New XKit Team</string>
 	<key>Builder Version</key>
-	<string>10600.1.25</string>
+	<string>10601.1.56.2</string>
 	<key>CFBundleDisplayName</key>
-	<string>XKit</string>
+	<string>New XKit</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.studioxenix.xkit</string>
+	<string>com.xkitplus.xkit</string>
```

Comment:
Rebase made GH hid my comment, so here it is again:

`xkitplus` vs `new[-]xkit`?